README.md

```diff
@@ -61,14 +81,77 @@ netflix.getRatingHistory(function (error, ratings) {
 })
 ```
 
+### Get Viewing History
+```javascript
+netflix.getViewingHistory(function (error, viewing) {
+  ratings === [
```

Comment:
To me, this is really confusing. I understand that you want to tell the user what the response will look like, but ratings is not defined in the code sample and besides that, I would rather have that information outside of the code segment. You could add a paragraph below the code explaining that the callback's second parameter `viewingHistory` (IMO `viewing` does not describe the data well) will contain data formated as so...

Reply:
Totally right, I'm simply logging the result now.
Let me know what you think